models/MARBL_joint_estimation/work/input.nml

```diff
@@ -0,0 +1,242 @@
+&perfect_model_obs_nml
+   read_input_state_from_file = .true.,
+   single_file_in             = .false.,
+   input_state_files          = "/glade/work/rarmstrong/cesm/cesm2_3_alpha12b+mom6_marbl/components/mom/standalone/examples/single_column_MARBL/BATS_joint_estimation/RESTART/DART_perfect_model_obs/MOM.res.nc",
+
+   write_output_state_to_file = .true.,
+   single_file_out            = .false.,
+   output_state_files         = "/glade/work/rarmstrong/cesm/cesm2_3_alpha12b+mom6_marbl/components/mom/standalone/examples/single_column_MARBL/BATS_joint_estimation/RESTART/DART_perfect_model_obs/MOM.res.output.nc",
+   output_interval            = 1,
+
+   async                      = 0,
+   adv_ens_command            = "./advance_model.csh",
+
+   obs_seq_in_file_name = "obs_seq.in",
+   obs_seq_out_file_name = "obs_seq.out",
+   init_time_days             = 0,
+   init_time_seconds          = 0,
+   first_obs_days             = -1,
+   first_obs_seconds          = -1,
+   last_obs_days              = -1,
+   last_obs_seconds           = -1,
+
+   trace_execution            = .false.,
+   output_timestamps          = .false.,
+   print_every_nth_obs        = -1,
+   output_forward_op_errors   = .false.,
+   silence                    = .false.,
+   /
+
+&filter_nml
+   single_file_in               = .false.,
+   input_state_files = '',
+   input_state_file_list = 'states_input.txt', 'params_input.txt',
```

Comment:
> If I set `estimate_params = .false.` in the `model_nml`, the code will still look for a list of parameter files. I guess that's ok given that this is a joint state-parameter estimation routine. This can be easily fixed once we work on merging the 3 different filtering strategies together.

merging the 3 different filtering strategies is happening as part of this pull request, right?

Reply:
Helen, I'm not quite sure. I'm happy to do it but it would take some time as I'm leaving. I say, we push the joint one now after small corrections. Once I'm back, I can get to it and add flexibility to perform the 3 using one code.

Reply:
There is quite a bit to do before releasing on this code.  I'll go ahead an close the pull request.  We can reopen it when it its ready.